app/models/spree/gateway/stripe_connect.rb

```diff
@@ -127,6 +134,11 @@ def tokenize_instance_customer_card(customer, card)
         Rails.logger.error("Stripe Error: #{e}")
         nil
       end
+
+      def ensure_enterprise_selected
+        return if preferred_enterprise_id.andand > 0
```

Comment:
I don't quite understand what you check with `andand`. It should be enough with `return if preferred_enterprise_id.present?`

Reply:
As above, it's a function of the way the preference is stored. I've specified it an as integer, and so it is defaulting to 0. `0.present?` is true, and I don't want to return if the value is 0. 

I will investigate whether I can specify a default of `nil`, and avoid all this fluffing around. 🙂

Reply:
@sauloperez I have tried but I can get `preferred_enterprise_id` to default to nil. It seems that when a `:integer` is specified as the type the default is always `0`, even if I try `default: nil`. I think that unfortunately I am going to have to keep this use of `andand`. Unless you can think of a better way of doing it?